test/testapps/BasicTestApp/InteropTest/JavaScriptInterop.cs

```diff
@@ -12,42 +13,41 @@ public class JavaScriptInterop
     {
         public static IDictionary<string, object[]> Invocations = new Dictionary<string, object[]>();
 
+        [JSInvokable(nameof(ThrowException))]
         public static void ThrowException() => throw new InvalidOperationException("Threw an exception!");
 
-        public static Task AsyncThrowSyncException() => throw new InvalidOperationException("Threw a sync exception!");
+        [JSInvokable(nameof(AsyncThrowSyncException))]
+        public static Task AsyncThrowSyncException()
+            => throw new InvalidOperationException("Threw a sync exception!");
 
-        public static Task AsyncThrowAsyncException()
+        [JSInvokable(nameof(AsyncThrowAsyncException))]
+        public static async Task AsyncThrowAsyncException()
         {
-            TaskCompletionSource<int> tcs = new TaskCompletionSource<int>();
-            var timer = new Timer(
-                state =>
-                {
-                    tcs.SetException(new InvalidOperationException("Threw an async exception!"));
-                },
-                null,
-                3000,
-                Timeout.Infinite);
-
-            return tcs.Task;
+            await Task.Delay(300);
```

Comment:
You can also use `Task.Yield()` here if you want to avoid a slowing down the test significantly. That's the documented way to actually force asynchrony

Reply:
Good tip - done.